src/settings/ServicePoints/ServicePointForm.js

```diff
@@ -174,6 +200,14 @@ const ServicePointForm = ({
     )));
   };
 
+  const handleChange = (e) => {
+    if (isConfirmPickupLocationChangeModalShouldBeVisible(search, e.target.value)) {
```

Comment:
`e.target.value` can be moved to separate var, because used twice in func

Reply:
Fixed